eth_account/_utils/transaction_utils.py

```diff
@@ -104,7 +107,9 @@ def transaction_rlp_to_rpc_structure(dictionary: Dict[str, Any]) -> Dict[str, An
     return dictionary
 
 
-def _access_list_rlp_to_rpc_structure(access_list: Sequence) -> Sequence:
+def _access_list_rlp_to_rpc_structure(
+    access_list: Sequence[Any],
```

Comment:
It looks like this is a pretty specific structure based on `t[0]` and `t[1]` from line 117. Looks like a two-tuple with address as the first and storage keys as the second. Maybe something like:
```suggestion
    access_list: Tuple[Address, Sequence[Bytes32]]
```

Reply:
We're mostly using `HexStr` for addresses here. I'll be taking another look at all this to see if the `TxData` typed dict from web3 could be shared with eth-account too.